docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -191,6 +191,31 @@ tasks.named('compileKotlin', KotlinCompilationTask) {
 
 Also, see [Types for compiler options](#types-for-compiler-options).
 
+#### Example of setting a languageVersion
```

Comment:
Should we also provide an example how to set `apiVersion`? :thinking:

Reply:
I think that is not necessary; the main idea is to have code snippets with the languageVersion setting and to give a link to this example in What's new.